app/src/main/java/com/hammer/talkbbokki/TalkbbokkiApp.kt

```diff
@@ -35,4 +42,18 @@ class TalkbbokkiApp : Application() {
             }.start()
         }
     }
+
+    private fun initNotificationChannels() {
+        val notificationManager =
+            getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager?
+        notificationManager ?: return
+        if (notificationManager.getNotificationChannel(NOTIFICATION_CHANNEL_ID) == null) {
+            val channel = NotificationChannel(
+                NOTIFICATION_CHANNEL_ID,
+                getString(R.string.menu_setting_push),
+                NotificationManager.IMPORTANCE_DEFAULT
+            )
+            notificationManager.createNotificationChannel(channel)
+       }
+   }
```

Comment:
🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (3) (should be 4)